tests/test_linux_arm.py

```diff
@@ -26,12 +26,12 @@
 
 
 class ZelosTest(unittest.TestCase):
-    def test_static_elf_verbose(self):
+    def test_static_elf_code_feed(self):
         z = Zelos(
             path.join(DATA_DIR, "static_elf_arm_helloworld"),
-            tracethread="none",
+            shutup=True,
+            code_feed="",
```

Comment:
code_feed?

Reply:
Annnnnddd yeah you're right. It's an inst_feed